webapp/pages/terms-and-conditions.vue

```diff
@@ -1,21 +1,12 @@
 <template>
-  <div>
-    <ds-space margin="small">
-      <ds-heading tag="h2">{{ $t('site.termsAndConditions') }}</ds-heading>
-    </ds-space>
-    <ds-container>
-      <div v-html="$t('html.termsAndConditions')" />
-    </ds-container>
-  </div>
+  <internal-page :pageParams="pageParams" />
 </template>
 
 <script>
+import links from '~/constants/links.js'
+import { internalPageMixins } from '~/mixins/internalPageMixins'
+
 export default {
-  layout: 'basic',
-  head() {
-    return {
-      title: this.$t('site.termsAndConditions'),
-    }
-  },
+  mixins: [internalPageMixins(links.TERMS_AND_CONDITIONS)],
 }
 </script>
```

Comment:
This template repeats - can we make one component out of this and register the url path somehow in vue instead of creating the component over and over again?

Reply:
@ulfgebhardt will create a separate issue … 😍